flink-runtime/src/test/java/org/apache/flink/runtime/shuffle/ShuffleMasterTest.java

```diff
@@ -99,7 +95,7 @@ public void testStopTrackingPartition() throws Exception {
                     EXTERNAL_PARTITION_RELEASE_EVENT,
                     EXTERNAL_PARTITION_RELEASE_EVENT,
                 };
-        assertArrayEquals(expectedPartitionEvents, TestShuffleMaster.partitionEvents.toArray());
+        assertThat(TestShuffleMaster.partitionEvents.toArray()).isEqualTo(expectedPartitionEvents);
```

Comment:
Possible Assert4j shorthand:
```java
        assertThat(TestShuffleMaster.partitionEvents)
                .containsExactly(
                        PARTITION_REGISTRATION_EVENT,
                        PARTITION_REGISTRATION_EVENT,
                        PARTITION_REGISTRATION_EVENT,
                        PARTITION_REGISTRATION_EVENT,
                        EXTERNAL_PARTITION_RELEASE_EVENT,
                        EXTERNAL_PARTITION_RELEASE_EVENT);
```

Reply:
Thanks for pointing that, I've prepared an improvement commit